tests/integration/helpers.py

```diff
@@ -436,9 +455,15 @@ async def wait_for_mongodb_units_blocked(
 
     This is necessary because the MongoDB app can report a different status than the units.
     """
-    for attempt in Retrying(stop=stop_after_delay(timeout), wait=wait_fixed(1), reraise=True):
-        with attempt:
-            await check_all_units_blocked_with_status(ops_test, db_app_name, status)
+    hook_interval_key = "update-status-hook-interval"
+    try:
+        old_interval = (await ops_test.model.get_config())[hook_interval_key]
+        await ops_test.model.set_config({hook_interval_key: "1m"})
```

Comment:
QQ I assume 1m is the shortest period of time. Is that correct?

Reply:
It could be lower (eg `15s`) but I noticed that it sometimes would leave units in state `executing` forever because it would get overwhelmed by `update status` events so I rather went for a safe in-between

Reply:
hmm unless the old interval is lower than 1m, this looks like a hack 
@Gu1nness What causes the avalanche of update-status events? I would rather we investigate that and fix the root cause of wrongly triggered events